pkg/models/datasource.go

```diff
@@ -144,9 +144,9 @@ type AddDataSourceCommand struct {
 	IsDefault         bool              `json:"isDefault"`
 	JsonData          *simplejson.Json  `json:"jsonData"`
 	SecureJsonData    map[string]string `json:"secureJsonData"`
-	ReadOnly          bool              `json:"readOnly"`
 
-	OrgId int64 `json:"-"`
+	OrgId    int64 `json:"-"`
+	ReadOnly bool  `json:"-"`
```

Comment:
Why do we keep the `ReadOnly` field for `AddDataSourceCommand` and `UpdateDataSourceCommand`? I assume these structs define the inputs for those command, so a ReadOnly field wouldn't be used (same goes for OrgId)?

Reply:
The reason for keeping `ReadOnly` is that we only want `ReadOnly` to be ignored when the datasource is created/updated via the API but not for example via provisioning.